honeybee_doe2/reader.py

```diff
@@ -0,0 +1,26 @@
+import os
+from math import radians, sin, cos
+from typing import Tuple, List
+from ladybug_geometry.geometry3d import Point3D, Face3D
+from ladybug_geometry.geometry2d import Point2D
+from ladybug_geometry.geometry3d import Plane, Vector3D
+from .util import parse_inp_file      
+from .config import DOE2_ANGLE_TOL, GEO_DEC_COUNT
+
+def model_from_inp(inp_file_path):
```

Comment:
We might just want to change this to `structured_dict_from_inp` for now just to make sure that no one accidentally builds something on top of this function that does not yet return a Honeybee Model.